Flutter/shelter_partner/lib/views/pages/enrichment_page.dart

```diff
@@ -426,6 +426,8 @@ class _EnrichmentPageState extends ConsumerState<EnrichmentPage>
     final appUser = ref.watch(appUserProvider);
     final shelterSettings = ref.watch(shelterSettingsViewModelProvider);
     final accountSettings = ref.watch(accountSettingsViewModelProvider);
+    selectedLocationTierCount =
+        accountSettings.value?.accountSettings?.locationTierCount ?? 2;
```

Comment:
Current default is 4 so I'd assume stick with that here as well unless you've heard otherwise
```suggestion
        accountSettings.value?.accountSettings?.locationTierCount ?? 4;
```